index.js

```diff
@@ -39,4 +41,101 @@ server.get("/api/posts/:id", (req, res) => {
     );
 });
 
+server.post("/api/posts", (req, res) => {
+  const postInfo = req.body;
+  console.log(req.body);
+  if (!postInfo.title || !postInfo.contents) {
+    res.status(400).json({
+      errorMessage: "Please provide title and contents for the post."
+    });
+  } else {
+    db.insert(postInfo)
+      .then(resultId => {
+        console.log(resultId);
+        db.findById(resultId.id)
```

Comment:
It might be better to call this something else or just go ahead and destructure the id off, i.e.:
```javascript
db.insert(postInfo)
  .then(({ id }) => {
     console.log(id);
     db.findById(id)
     // ...
```